new-lamassu-admin/src/components/date-range-picker/Tile.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react'
+import classnames from 'classnames'
+import { makeStyles } from '@material-ui/core/styles'
+import { primaryColor, spring2, spring3, fontSecondary, disabledColor } from '../../styling/variables'
+
+const styles = {
+  wrapper: {
+    width: 45,
+    height: 26,
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    position: 'relative'
+  },
+  button: {
+    border: 'none',
+    width: '100%',
+    height: '100%',
+    cursor: 'pointer',
+    padding: 0,
+    backgroundColor: 'transparent',
+    color: primaryColor,
+    zIndex: 2,
+    fontFamily: fontSecondary,
+    fontSize: 14,
+    fontWeight: 500
+  },
+  lowerBound: {
+    width: [['50%', '!important']],
```

Comment:
using !important should be prevented, I'll take a look at why you're needing to do this and detail this comment further.

Reply:
It's so it doesn't overflow when the bounds are on Saturdays/Sundays.
![image](https://user-images.githubusercontent.com/5789328/68760370-973c3880-0609-11ea-9472-527029356af7.png)
I can replace it with an `overflow: hidden` on the wrapper, if that's preferred.

Reply:
I also noticed a typo, should be `[['50%'], '!important']`.

Reply:
Yeah, if overflow hidden on the wrapper works that's preferred.